tests/app/legacy/dao/conftest.py

```diff
@@ -11,7 +11,7 @@
 
 
 @pytest.fixture
-async def prepared_service(
+async def commit_service(
     sample_service: Callable[[async_scoped_session[AsyncSession]], Awaitable[Row[Any]]],
 ) -> AsyncGenerator[Row[Any], None]:
     """Fixture that creates, commits, and yields a sample service row for integration tests.
```

Comment:
I know you didn't write this, but should this be using the `test_db_session` fixture?

Reply:
The `test_db_session` was intended for non-commit sessions, but it's been used to commit and not commit. After this gets approved I'll do a refactor in a separate PR. We'll call it `no_commit_session` and I'll change any that commit with it to not do that.

Only two cases to commit to the DB while testing:

1. DAO methods
2. `client.post`, `client.get`, etc.